internal/receiver/databricksreceiver/factory.go

```diff
@@ -0,0 +1,85 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package databricksreceiver
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/config"
+	"go.opentelemetry.io/collector/consumer"
+	"go.opentelemetry.io/collector/receiver/receiverhelper"
+	"go.opentelemetry.io/collector/receiver/scraperhelper"
+)
+
+const typeStr = "databricks"
+
+func NewFactory() component.ReceiverFactory {
+	return receiverhelper.NewFactory(
+		typeStr,
+		createDefaultConfig,
+		receiverhelper.WithMetrics(createReceiverFunc(newAPIClient)),
+	)
+}
+
+type Config struct {
+	InstanceName                            string `mapstructure:"instance_name"`
+	BaseURL                                 string `mapstructure:"base_url"`
+	Token                                   string
+	scraperhelper.ScraperControllerSettings `mapstructure:",squash"`
+}
+
+func createDefaultConfig() config.Receiver {
+	return &Config{
+		ScraperControllerSettings: scraperhelper.ScraperControllerSettings{
+			ReceiverSettings:   config.NewReceiverSettings(config.NewComponentID(typeStr)),
+			CollectionInterval: time.Second * 30,
+		},
+	}
```

Comment:
```suggestion
	scs := scraperhelper.DefaultScraperControllerSettings(typeStr)
	scs.CollectionInterval = 30 * time.Second
	
	return &Config{
		ScraperControllerSettings: scs,
	}
```
This could maybe help if scraper settings change in the future. Also, is the default of `30s` something specific to databricks or just an arbitrary number? I would probably leave as default for consistency with other scrapers otherwise.

Reply:
Thanks, fixed. I set it to 30s because that's double the highest job frequency of 1/minute -- added a comment.